pages/account/Dashboard.js

```diff
@@ -0,0 +1,42 @@
+// friends
+// Pull in timezones from a json file and display them:
+// import
+import styles from '../../styles/dashboard.module.css'
+import AddFriend from '../../components/AddFriend';
+
+const UTC_OFFSETS = require('/data/timezones.json');
+const timeSettings = { hour: '2-digit', minute: '2-digit' };
+
+function DisplayOffsets(props) {
+    const offsets = props.offsets
+    // Display offsets
+    const offsetDisplays = offsets.map((offset) =>
```

Comment:
We can return the expression here immediately instead of assigning it to a temporary variable. Could you move DisplayOffsets to its separate component, pls?